crates/derive/Cargo.toml

```diff
@@ -15,9 +15,10 @@ anyhow = "*"
 bytes = "*"
 futures = "*"
 itertools = "*"
+librocksdb-sys = { version = "*", features = ["lz4", "rtti"] }
 pin-utils = "*"
 prost = "*"
-rocksdb = { version = "*", default-features = false }
+rocksdb = { version = "*", default-features = false, features = ["lz4", "rtti"] }
```

Comment:
Do we need to specify the features on both of these libraries explicitly? Would specifying them on `rocksdb` alone apply them to `librocksdb-sys` as well?